### PR TITLE
Bootstrap smallboxes

### LINKED ISSUE #1875: when screen is a large width, seach field gets small
screen shot 2016-04-15 at 11 15 36 am

It looks okay at a medium width.

### PR DESCRIPTION
**Pull request by atz**
fixes #1875 

Consolidates feature to use the same CC template.  Delivers more uniform bootstrap styling.

### CODE REVIEW

File: app/views/collections/edit.html.erb [hunk 1]
```diff
@@ -16,8 +16,14 @@
   </div>
 <% end %>
 
-<h2>Manage Items in this Collection</h2>
-<%= render 'search_collection_dashboard_form'%>
+<br/>
```
Comment (cbeer) [inline app/views/collections/edit.html.erb]: `<br />`? Can we style things appropriately instead?

File: app/views/collections/edit.html.erb [hunk 2]
```diff
@@ -16,8 +16,14 @@
...
+<br/>
+<div class="row">
+  <h2 class="col-xs-12 col-sm-6"><%= t('Manage Items in this Collection') %></h2>
```
Comment (cbeer) [inline app/views/collections/edit.html.erb]: Does this `#t` actually work?

Comment (atz) [inline app/views/collections/edit.html.erb]: Yep.  I don't have a populated translation, but it gives me back the parameter string at least.

**Comment by atz** [comment]
Rebased.

**Comment by mjgiarlo** [comment]
@atz Inclined to make the changes @cbeer suggests above?

File: app/views/collections/edit.html.erb (hunk 2 above)
Comment (cbeer) [inline app/views/collections/edit.html.erb]: I'd guess it also gives a `span.missing_translation`, and might be difficult to make work in the i18n file with all the spaces (and the lack of expected namespacing)?

**Comment by atz** [comment]
There was no such warning delivered.  If it is important, there probably should be one.  

But I agree using the translation file is more proper.  Pushed modified branch.

**Comment by atz** [comment]
As for `<br/>`, I don't care.  It's not worth holding anything up about.  I know we aren't supposed to mix bootstrap and inline styles, so I attempted to find the correct bootstrapism and felt like it was a wasted effort.  Vertical separator was removed in bootstrap 3.  _Maybe_ something will be included for [bootstrap 4](https://github.com/twbs/bootstrap/issues/4286#issuecomment-36331550).  

If you want to declare what "appropriate style" would be, I can transcribe it for you.  Or in the same amount of time, you could push to this topic branch.

**Comment by cbeer** [comment]
What are we trying to do with the `<br />` anyway? Does it actually look bad without them?

A far preferable solution, in my mind, may involve adding a class or id to an element (the `h2`, or whatever element we're trying to create a margin from) and add explicit styling for it.

**Comment by atz** [comment]
Yeah, it looks bad w/o some separation.  Adding an ID is unnecessary, since presumably the need for vertical separation is not unique to a single element.  What class makes sense?  `.edit_collection`?  That provides:
```scss
margin-top: 2 * $padding-large-vertical;
```

And also doesn't appear to be used anywhere in the application.

**Comment by cbeer** [comment]
I was assuming a bespoke class, named appropriately for whatever we're trying to style.

File: app/views/collections/_sort_and_per_page.html.erb [hunk 3]
```diff
@@ -3,12 +3,12 @@
     <%= render partial: 'collections/form_for_select_collection', locals: { user_collections: @user_collections }  %>
   </div>
 
-  <% if  params[:action] == "edit" %>
+  <% if params[:action] == "edit" && @response.response['numFound'] > 1 %>
```
Comment (mjgiarlo) [inline app/views/collections/_sort_and_per_page.html.erb]: Can you say more about this new condition?

Comment (atz) [inline app/views/collections/_sort_and_per_page.html.erb]: When there is nothing to display, don't attempt to display it.

Comment (cbeer) [inline app/views/collections/_sort_and_per_page.html.erb]: How about `any?` or `many?`?

**Pull request merged**